tools/previous-best/best_solvers_to_csv.py

```diff
@@ -0,0 +1,154 @@
+            help="The registration csv file where the winners are from")
+    required.add_argument("-n", "--new-registration",
+            action="store",
+            dest="new_registration",
+            required=True,
+            help="The registration csv file where the previous winners"\
+                "would be placed")
+    required.add_argument("-y", "--old-year",
+            action="store",
+            dest="old_year",
+            required=True,
+            type=int,
+            help="The old result year (for renaming entries)")
+
+
+    g_args = parser.parse_args()
+
+if __name__ == '__main__':
+    global g_args
+    parse_args()
+
+    winner_files= [g_args.sq_winners, g_args.uc_winners, \
+            g_args.in_winners, g_args.mv_winners]
+
+    winners = {}
+
+    for i in range(0, len(winner_files)):
+        f = winner_files[i]
+        with open(f) as csvfile:
+            reader = csv.DictReader(csvfile)
+            for row in reader:
+                name = row[WINNER_NAME_COL]
+                div = row[WINNER_DIV_COL]
+                if name not in winners:
+                    winners[name] = \
+                            [[] for k in range(0, len(winner_files))]
+                assert(i < len(winners[name]))
+                winners[name][i].append(div)
+
+    new_header = None
+    with open(g_args.new_registration) as new_registration:
+        new_registration = csv.DictReader(new_registration)
+        new_header = next(new_registration)
+
+    with open(g_args.old_registration) as old_registration:
+        old_registration = csv.DictReader(open(g_args.old_registration))
+
+        out = open(g_args.outfile, 'w')
+        out.write("%s\n" % ",".join(list(new_header.keys())))
+        for row in old_registration:
+            if row[REG_NAME_COL] not in winners:
+                continue
+            # this is a winner row
+            name = row[REG_NAME_COL]
+            new_winner_row = []
+            for col in new_header:
+                if col in row:
+                    if col == REG_SQDIVS_COL:
+                        new_winner_row.append(";".join(winners[name][0]))
+                    elif col == REG_UCDIVS_COL:
+                        new_winner_row.append(";".join(winners[name][1]))
+                    elif col == REG_INDIVS_COL:
+                        new_winner_row.append(";".join(winners[name][2]))
+                    elif col == REG_MVDIVS_COL:
+                        new_winner_row.append(";".join(winners[name][3]))
+                    elif col == REG_COMPETING_COL:
+                        new_winner_row.append("no")
+                    elif col == REG_SEED_COL:
+                        new_winner_row.append("")
+                    elif col == REG_NAME_COL:
+                        new_winner_row.append(adjustName(name, \
+                            g_args.old_year))
+                    else:
+                        new_winner_row.append(quoteIfHasComma(row[col]))
```

Comment:
Should we remove the contents of all columns except for the wrapped solver ids?

Reply:
Also @HanielB brought this up.  My justification of keeping them is that maybe an occasional browser of the website would like to see this information.  Pointing out by these empty fields that this is not an actual submission seems a little too subtle anyway.